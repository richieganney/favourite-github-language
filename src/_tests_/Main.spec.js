import React from "react";
import Main from "../Main"
import { create } from "react-test-renderer";

describe("Main component", () => {
  test("Matches the snapshot", () => {
    const main = create(<Main />);
    expect(main.toJSON()).toMatchSnapshot();
  });
});