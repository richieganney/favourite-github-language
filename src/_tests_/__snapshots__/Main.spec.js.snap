// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main component Matches the snapshot 1`] = `
<div>
  <div>
    <form
      id="search_bar"
    >
      <div
        className="search"
      >
        <input
          className="round"
          id="username_search_bar"
          onChange={[Function]}
          type="text"
          value=""
        />
      </div>
    </form>
  </div>
</div>
`;
